app/assets/javascripts/discourse/templates/discovery/categories.hbs

```diff
@@ -22,16 +22,16 @@
                 <div class="category-description">
                   {{{c.description_excerpt}}}
                 </div>
+                {{#if c.category_expanding_class}}
+                    <button class="btn"
+                      {{action "expand" c.category_expanding_class}}
+                      {{bind-attr id="c.category_expanding_class"}}
+                    >
+                        Expand units for this course
+                    </button>
+                {{/if}}
```

Comment:
No -`category_expanding_class` gets set if you are the parent (slug name is csp or csd). `category_collapse_class` gets set if youare the child (slug name begins with csp or csd and has numbers after it)